spec/models/activity_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe Activity, type: :model do
+  describe "Validations" do
+    describe "#activity_type" do
+      it "does not allow nil for activity_type"
+      #set to nil
```

Comment:
so, here, be sure to add the do at the end of the line. I left it out because it's another way of leaving a test pending.

So, for this we can do a few things...we can install shoulda_matchers gem. It's pretty great and has a lot of helpers for testing validations, or we can continue down the path of using vanilla rspec.

Let's pair on this tomorrow, and let's plan on meeting Wednesday too.